source/declarations.tex

```diff
@@ -5324,8 +5327,8 @@
 functions\iref{over.match.ref} and choosing the best one through overload
 resolution\iref{over.match}),
 \end{itemize}
-then the reference is bound to the initializer expression lvalue in the
-first case and to the lvalue result of the conversion
+then the reference is bound to the object or function to which the initializer expression refers in the
+first case and to the object to which the lvalue result of the conversion refers
```

Comment:
The "lvalue result of the conversion" here was already correct; see the definition of "result" in [basic.lval]p5. I think we could use "result of" more broadly in this change. Eg:

"the reference is bound to the lvalue result of the initializer expression in the first case and to the lvalue result of the conversion in the second case".

Reply:
Looks good. One gripe I have with it is that when we say "glvalue" result, it could be interpreted in one of two ways, the resulting glvalue of the conversion (see my other comment as to why conversions result in expressions), or the result of the glvalue. It would be best to make it as clear as possible to avoid any confusion.